src/simulations/explorer.py

```diff
@@ -944,6 +963,7 @@ def _record_last_epoch_opt(
             break_down_rewards,
             loaded_buses, sgen_buses
         ]
+        record_output.append(res['q_ext_grid'][time_step])
```

Comment:
this can go in the list above; or you can move `q_ext_grid` right before/after `hourly_line_losses` in the `last_entries` list, so you can add it in a loop using the same format.